.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+cache: pip
+python:
+    - 2.7
```

Comment:
I think the python-miio lib only works with python3 so 2.7 should probably be disabled for the CI? see https://github.com/rytilahti/python-miio/blob/master/.travis.yml